src/LIBSVM.jl

```diff
@@ -217,6 +234,38 @@ function check_train_input(X, y, kernel)
     end
 end
 
+function data2gram(kernel_function::Function, X)
+    n1 = axes(X, 2)
+    gram = Array{Float64}(undef, n1.stop, n1.stop)
+    for i in n1
+        for j in 1:i
+            if i==j
+                gram[i,i] = kernel_function(X[:,i], X[:,i])
+            else
+                interm = kernel_function(X[:, i], X[:, j])
+                gram[i,j] = interm
+                gram[j,i] = interm
+            end
```

Comment:
Would it be too bad to do
```julia
gram[i, j] = gram[j, i] = kernel_function(X[:, i], X[:, j])
```
?

Reply:
I would be fine with this. I'm pretty new to Julia, is there a standard way of handling a case like this?

Reply:
I guess we could consider this a standard way.